project/api/serializers.py

```diff
@@ -52,6 +52,18 @@ def to_internal_value(self, data):
 
         return ret
 
+    def to_representation(self, instance):
+        """
+        Serialize territories for API response.
+        """
+        data = super(TerritorySerializer, self).to_representation(instance)
+
+        # Compress geojson to geobuf and return as hexadecimal
+        gbuf = geobuf.encode(data["geo"])
```

Comment:
Since we are only compressing a single field, consider refactoring this code into a single field (https://www.django-rest-framework.org/api-guide/relations/#custom-relational-fields). This may or may not be worth the effort, so only do this if you can convince yourself it's worth it.